src/buttons/Chip.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { TouchableWithoutFeedback, StyleSheet } from 'react-native';
+import { withTheme } from '../config';
+import Button, { ButtonProps } from './Button';
+
+export type ChipProps = Omit<
+  ButtonProps,
+  'loading' | 'loadingStyle' | 'loadingProps'
+> & {
+  type?: 'solid' | 'outline';
+};
+
+const Chip: React.FunctionComponent<ChipProps> = (props: ChipProps) => {
+  return (
+    <Button
+      titleStyle={StyleSheet.flatten([
+        props.titleStyle,
+        { fontSize: 14, paddingHorizontal: 2 },
```

Comment:
This is the same issue. Here, someone won't be able to overwrite `fontSize` or `paddingHorizontal`. the `props.titleStyle` has to come last.

Reply:
Ahh okay, I see what you mean. I misunderstood how StyleSheet.flatten works (anything we add should come before what the user passes in, got it). Thanks! I've updated the props! Kindly review and let me know if there are any other issues I've overlooked.